### PR TITLE
Add country and venue columns

### LINKED ISSUE #41: Separating Location field
I'm working on a project to automate listing the upcoming Python-related conferences and I'd like to use this repo as a source of data. On thing I'd like to be able to do is filter / separate conferences by region or country. With the current "Location" field it's a little hard to do. Would you be open to separating the field into "City", "Region", "Country" (maybe with "Venue" as well?) or adding those as new fields?

Thanks for maintaining this!

### PR DESCRIPTION
**Pull request by tylerdave**
* Add `Country` column with data for all events
* Add `Venue` column with some data (the ones I knew offhand)
* Updates README to include a description of the fields

Resolves #41

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,24 @@
...
+|-------------------|--------------------------------------------|------------------------------------------------------------------------|-----------------------------------
+| Subject           | Event name                                 | Text                                                                   | required, used by Google Calendar
+| Start Date        | First day of event                         | YYYY-MM-DD                                                             | required, used by Google Calendar
+| End Date          | Last day of event                          | YYYY-MM-DD                                                             | used by Google Calendar
+| Location          | Location of the event                      | Text                                                                   | used by Google Calendar
+| Country           | Country code                               | [ISO 3166-1 alpha-3](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3) |
+| Venue             | Event venue                                | Text                                                                   |
```
Comment (Mariatta) [inline README.md]: It wasn't too clear at first the difference of Venue and Location. Perhaps the description should say "Venue Name", and the note can further explain:  "The name of the Venue, e.g.: Oregon Convention Center "

**Comment by Mariatta** [comment]
Oops, sorry @tylerdave I made a conflict in your PR by merging #72 :(

**Comment by tylerdave** [comment]
All good. I was going to update based on your comment above. I'll do that and resolve the conflict later today.

**Comment by tylerdave** [comment]
I _think_ it should be good now!

**Pull request merged**